roles/network_plugin/calico/tasks/main.yml

```diff
@@ -6,6 +6,13 @@
     enabled: yes
   failed_when: false
 
+- name: Calico | Get kubelet hostname
```

Comment:
Is this any more readable if we use uri module to hit the apiserver directly versus a shell command?

Reply:
I doubt it. I will implement this way first, then refactor using downward api for generating calico cni, as upstream does.